lib/commands/build.js

```diff
@@ -7,8 +7,8 @@ const StripesPlatform = importLazy('../platform/stripes-platform');
 const { okapiOptions, stripesConfigFile, stripesConfigStdin, stripesConfigOptions, buildOptions } = importLazy('./common-options');
 const { processError, processStats, emitLintWarnings, limitChunks } = importLazy('../webpack-common');
 
-
-function buildCommand(argv) {
+// stripesPlatform and stripesCore are optional params primarily used as injection for unit tests
+function buildCommand(argv, stripesPlatform, stripesCore) {
```

Comment:
Not entirely sure about the precedent this will set given these command handlers are adhering to an interface invoked by Yargs.  For instance, this could get difficult to maintain should a future version of Yargs pass additional parameters to command handlers.

Reply:
Good point. I could switch to an optional constructor or a public setter. I originally had tried to mock the imports with `sinon` but couldn't get it to work possibly because of the lazy import.

Reply:
FWIW, I'm not too sure how rigorous we need to be about testing the command handlers themselves anyway.  Ideally the handlers don't contain much logic, only handling input and output messaging.  They are primarily calling other functions to do work (that are themselves more readily tested).